src/view/com/notifications/FeedItem.tsx

```diff
@@ -237,7 +240,7 @@ let FeedItem = ({
                   <Trans>and</Trans>{' '}
                 </Text>
                 <Text style={[pal.text, s.bold]}>
-                  {formatCount(authors.length - 1)}{' '}
+                  {FormatCount(authors.length - 1)}{' '}
```

Comment:
Why was this function capitalized?

Reply:
I changed it to capitalized because I got a linting error after localization, if there's another way to do it, I'd appreciate it.

Reply:
I resolve the linting error. It reverted to lowercase.